src/htm/algorithms/TemporalMemory.cpp

```diff
@@ -515,7 +515,7 @@ void TemporalMemory::compute(const SDR &activeColumns,
                          cellsToColumns( getPredictiveCells() ));
       const Real like = tmAnomaly_.anomalyLikelihood_.anomalyProbability(raw);
       const Real log  = tmAnomaly_.anomalyLikelihood_.computeLogLikelihood(like);
-      tmAnomaly_.anomaly_ = log;
+      tmAnomaly_.anomaly_ = log; //TODO loglike is `log(1 - likelihood)/log(epsilon)`, should we return 1-loglike to be consistent with "anomalies"? (high score for anomalous inputs).
```

Comment:
@thanh-binh Thank you for your experiments with Likelihood! 
Could you please review that this doc is more readable and explains the issue? 
Also, is it correct that your problem was seen for "LOGLIKELIHOOD" mode? In that case, should we return `1-log` on this line, to be consistent with anomaly?